pkg/sql/create_type.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package sql
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/util/errorutil/unimplemented"
+	"github.com/cockroachdb/errors"
+)
+
+type createTypeNode struct {
+	n *tree.CreateType
+}
+
+// Use to satisfy the linter.
+var _ planNode = &createTypeNode{n: nil}
```

Comment:
if you felt poncy. `var _ planNode = (*createTypeNode)(nil)`

Reply:
nah i need to use `n` LOL

Reply:
wow the go compiler is rood.